### PR TITLE
client: add Node.TTLDuration()

### PR DESCRIPTION
**Pull request by bcwaldon**

### CODE REVIEW

File: client/keys.go [hunk 1]
```diff
@@ -274,6 +274,14 @@ func (n *Node) String() string {
 	return fmt.Sprintf("{Key: %s, CreatedIndex: %d, ModifiedIndex: %d, TTL: %d}", n.Key, n.CreatedIndex, n.ModifiedIndex, n.TTL)
 }
 
+// TTLDuration returns the Node's TTL as a time.Duration object
+func (n *Node) TTLDuration() time.Duration {
+	if n == nil {
```
Comment (xiang90) [inline client/keys.go]: hmm... when do you expect this to happen?

Comment (bcwaldon) [inline client/keys.go]: Just covering all possible cases. I'm more than happy not covering this if you prefer.

Comment (xiang90) [inline client/keys.go]: This will mask a panic case. (I assume the nil Node is a really bad case)

So I am not sure if it worth the effort.

**Comment by xiang90** [comment]
I think we can remove the nil checking. Then LGTM. Thanks!

**Comment by bcwaldon** [comment]
@xiang90 Thanks! For reference, this came from the etcd client in fleet; converting the TTL to a time.Duration is a very common operation there.

**Pull request merged**